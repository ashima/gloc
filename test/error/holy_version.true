File 0, line 2, col 0 - 12:
version must be first semantic token

File 0, line 1, col 0 - 19:
#error this is fine

