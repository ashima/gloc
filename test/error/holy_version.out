File 0, line 2, col 0 - 12:
version must be first semantic token

